WordPress/JetpackScreenshotGeneration/Info.plist

```diff
@@ -20,9 +20,5 @@
 	<string>????</string>
 	<key>CFBundleVersion</key>
 	<string>1</string>
-	<key>WIREMOCK_HOST</key>
-	<string>$(WIREMOCK_HOST)</string>
-	<key>WIREMOCK_PORT</key>
-	<string>$(WIREMOCK_PORT)</string>
```

Comment:
I removed the two `WIREMOCK_` settings from the bundle in favor of 1) having them hardcoded and 2) override-able via environment variable. More details in `WireMock.swift` below.